lastfmscrobbler/scrobble.sh

```diff
@@ -6,8 +6,13 @@ tentativeMetadata=$(echo '{ "command": ["get_property", "metadata"] }' | /usr/bi
 
 sleep 4
 
-USERNAME="" # CHANGE ME
-PASSWORD="" # CHANGE ME
+USERNAME='' # CHANGE ME
+
+KEYCHAIN_SERVICE='com.iina.lastfm'
+# Retrieve password from the keychain
+# Ensure you've added it first using `security add-generic-password -a "lastfmUsername" -s "com.iina.lastfm" -w "mypassword"`
```

Comment:
maybe this would be a good option here to get their username, store it and set the password secret?

https://sapegin.me/blog/show-gui-dialog-from-shell/